src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncTests.cs

```diff
@@ -3616,6 +3615,15 @@ public void AwaitInScriptExpression()
         }
 
         [Fact]
+        public void AwaitInScriptStatement()
```

Comment:
I think the test should be called something like `AwaitInGlobalStatementInScript`, since `await` worked before if appears in a method declaration. This is not blocking the merge though since the renaming, if necessary, should be done across the entire file.